docs/requirements.txt

```diff
@@ -2,5 +2,6 @@ numpy
 scipy
 Cython
 numba
+git+git://github.com/mohanwu/eigenpip.git
```

Comment:
I hope you generate this file (`requirements.txt`) automatically from repo using `pip freeze`, as shown [here](https://pip.pypa.io/en/stable/user_guide/#requirements-files).

Reply:
Update this to use PyPi version of eigenpip.